src/meds_etl/omop.py

```diff
@@ -258,38 +290,55 @@ def main():
 
     code_metadata = {}
 
-    for concept_file in get_table_files(args.src_omop, "concept"):
+    print("Generating metadata from OMOP `concept` table")
+    for concept_file in tqdm(get_table_files(args.src_omop, "concept")):
+        if args.verbose:
+            print(concept_file)
         with load_file(decompressed_dir, concept_file) as f:
-            concept = pl.read_csv(f.name)
+            # Read the contents of the `concept` table shard
+            concept: pl.DataFrame = pl.read_csv(f.name)
             concept_id = pl.col("concept_id").cast(pl.Int64)
             code = pl.col("vocabulary_id") + "/" + pl.col("concept_code")
+
+            # Convert the table into a dictionary
             result = concept.select(concept_id=concept_id, code=code, name=pl.col("concept_name")).to_dict(
                 as_series=False
             )
+
+            # Update our running dictionary with the concepts we read in from 
+            # the concept table shard
             concept_id_map |= dict(zip(result["concept_id"], result["code"]))
             concept_name_map |= dict(zip(result["concept_id"], result["name"]))
 
+            # Assuming custom concepts have concept_id > 2000000000 we create a
+            # record for them in `code_metadata` with no parent codes. Such a
+            # custom code could be eg `STANFORD_RACE/Black or African American`
+            # with `concept_id` 2000039197
             custom_concepts = (
                 concept.filter(concept_id > 2_000_000_000)
-                .select(code=code, description=pl.col("concept_name"))
+                .select(concept_id=concept_id, code=code, description=pl.col("concept_name"))
```

Comment:
Why are you pulling concept_id here?

Reply:
Makes debugging easier with a trivial amount of increased memory/time requirement, but I can remove if you prefer

Reply:
That's fine. We can keep it. I was just wondering.